docker/setup.cfg

```
@@ -0,0 +1,35 @@
+[metadata]
+name = deeplabcut-docker
+version = attr: deeplabcut_docker.__version__
+author = A. & M. Mathis Labs
+author_email=alexander@deeplabcut.org
```

```suggestion
author = A. & M. Mathis Labs
author_email=alexander@deeplabcut.org
maintainer = Steffen Schneider
maintainer_email = stes@hey.com
```

While `author` & `author_email` field are copied from the main package, I'm happy to list myself as the maintainer here (for updating the pypi package etc.) until the first stable version is out, etc.